Exception/src/MethodsCalculator.java

```diff
@@ -0,0 +1,36 @@
+import java.util.InputMismatchException;
+import java.util.Scanner;
+
+public class MethodsCalculator {
+    Scanner sc = new Scanner(System.in);
+    double value = 0;
+    boolean invalidValue = true;
+    String operation;
+
+    public double setValue() {
+        while (invalidValue) {
```

Comment:
It's always true, please avoid such loops

Reply:
I understand, but my program doesn't work without it, and I didn't find a better solution for it. So I didn't change it.

Reply:
OK